docs/Catalog/Admin.md

```diff
@@ -65,10 +70,78 @@ The resulting permission set is equivalent to a union of all permissions
 provided by the policies attached to that role.
 
 
+### Extending built-in roles
+
+The initial "Source=Custom" roles defined by Quilt (`ReadQuiltBucket`, `ReadWriteQuiltBucket`) are locked to AWS ARNs, and thus cannot be extended by Quilt policies.  If you want a "Source=Quilt" UI-managed role that automatically has access to all registered buckets, you need to first import those specific polices.
+
+
+#### A. Create a Source=Quilt role
+
+For concreteness, we will create a new `UserReadQuiltBucket` read-only role to which we can add Quilt policies.
+
+1. Login to your Quilt instance at, e.g. https://quilt.mycompany.com
+2. Click on "Admin Settings" in the upper right, under your Profile name
+3. Scroll down to the "Roles" section on the bottom
+4. Click "+" to define a new role
+5. Name it `UserReadQuiltBucket`
+6. Save
+
+You can also create a `UserReadWriteQuiltBucket` if you want grant full permissions
+
+#### B. Display existing policies in AWS Console
```

Comment:
For now let's eliminate the user finding and attaching Quilt policies managed by the stack. Please only document creating and attaching Quilt policies, made in Quilt, via the UI.

Reply:
@akarve Do you still feel strongly that this revision should NOT allow people to create new Source=Quilt roles that leverage the existing Read/ReadWrite Source=Custom policies?